src/algebra/geom_sum.lean

```diff
@@ -114,6 +146,25 @@ begin
   rw [← this, add_sub_cancel]
 end
 
+lemma commute_op_op_of_commute [has_mul α] {x y : α} (h : commute x y) : commute (op y) (op x) :=
+begin
+  dunfold commute,
+  dunfold semiconj_by,
+  rw [← op_mul, ← op_mul, h.eq]
+end
+
+lemma mul_neg_geom_sum₂_comm [ring α] {x y : α} (h : commute x y) (n : ℕ) :
```

Comment:
This and all the other lemmas in this file that take `commute x y` as an argument should be renamed from `something_comm` to `commute.something` to enable dot notation.

Reply:
Thanks for the suggestion. Done.